docs/ops/state_backends.md

```diff
@@ -124,7 +124,7 @@ RocksDBStateBackend is currently the only backend that offers incremental checkp
 ## Configuring a State Backend
 
 State backends can be configured per job. In addition, you can define a default state backend to be used when the
-job does not explicitly define a state backend.
+job does not explicitly define a state backend. Besides, state backend configured per-job will overwrite the default state backend configured in `flink-conf.yaml`
```

Comment:
Would it be enough to reword the existing docs to:

"
State backends can be configured per job. In addition, you can define a default state backend **in `flink-conf.yaml`** to be used when the job does not explicitly define a state backend.
"